src/main/java/edu/next/racing/domain/RacingGame.java

```diff
@@ -58,6 +55,7 @@ private void setWinnerPosition() {
     }
 
     public List<Car> getWinner() {
+        setWinnerPosition();
```

Comment:
`winnerPosition`은 우승자를 구할 때만 사용하는 값이라, `Car` 인스턴스의 변수가 아니여도 될 것 같아요.

```suggestion
        int winnerPosition = setWinnerPosition();
```